iceoryx_posh/test/moduletests/test_mepoo_chunk_settings.cpp

```diff
@@ -0,0 +1,397 @@
+{
+    constexpr uint32_t PAYLOAD_SIZE{42U};
+    constexpr uint32_t PAYLOAD_ALIGNMENT{128U};
+    constexpr uint32_t CUSTOM_HEADER_SIZE{64U};
+    constexpr uint32_t CUSTOM_HEADER_ALIGNMENT{4U};
+
+    auto sutResult =
+        ChunkSettings::create(PAYLOAD_SIZE, PAYLOAD_ALIGNMENT, CUSTOM_HEADER_SIZE, CUSTOM_HEADER_ALIGNMENT);
+    ASSERT_FALSE(sutResult.has_error());
+    auto& sut = sutResult.value();
+
+    EXPECT_THAT(sut.payloadSize(), Eq(PAYLOAD_SIZE));
+}
+
+TEST(ChunkSettings_test, payloadAlignmentReturnsCorrectValue)
+{
+    constexpr uint32_t PAYLOAD_SIZE{42U};
+    constexpr uint32_t PAYLOAD_ALIGNMENT{128U};
+    constexpr uint32_t CUSTOM_HEADER_SIZE{64U};
+    constexpr uint32_t CUSTOM_HEADER_ALIGNMENT{4U};
+
+    auto sutResult =
+        ChunkSettings::create(PAYLOAD_SIZE, PAYLOAD_ALIGNMENT, CUSTOM_HEADER_SIZE, CUSTOM_HEADER_ALIGNMENT);
+    ASSERT_FALSE(sutResult.has_error());
+    auto& sut = sutResult.value();
+
+    EXPECT_THAT(sut.payloadAlignment(), Eq(PAYLOAD_ALIGNMENT));
+}
+
+TEST(ChunkSettings_test, customHeaderSizeReturnsCorrectValue)
+{
+    constexpr uint32_t PAYLOAD_SIZE{42U};
+    constexpr uint32_t PAYLOAD_ALIGNMENT{128U};
+    constexpr uint32_t CUSTOM_HEADER_SIZE{64U};
+    constexpr uint32_t CUSTOM_HEADER_ALIGNMENT{4U};
+
+    auto sutResult =
+        ChunkSettings::create(PAYLOAD_SIZE, PAYLOAD_ALIGNMENT, CUSTOM_HEADER_SIZE, CUSTOM_HEADER_ALIGNMENT);
+    ASSERT_FALSE(sutResult.has_error());
+    auto& sut = sutResult.value();
+
+    EXPECT_THAT(sut.customHeaderSize(), Eq(CUSTOM_HEADER_SIZE));
+}
+
+TEST(ChunkSettings_test, customHeaderAlignmentReturnsCorrectValue)
+{
+    constexpr uint32_t PAYLOAD_SIZE{42U};
+    constexpr uint32_t PAYLOAD_ALIGNMENT{128U};
+    constexpr uint32_t CUSTOM_HEADER_SIZE{64U};
+    constexpr uint32_t CUSTOM_HEADER_ALIGNMENT{4U};
+
+    auto sutResult =
+        ChunkSettings::create(PAYLOAD_SIZE, PAYLOAD_ALIGNMENT, CUSTOM_HEADER_SIZE, CUSTOM_HEADER_ALIGNMENT);
+    ASSERT_FALSE(sutResult.has_error());
+    auto& sut = sutResult.value();
+
+    EXPECT_THAT(sut.customHeaderAlignment(), Eq(CUSTOM_HEADER_ALIGNMENT));
+}
+
+TEST(ChunkSettings_test, requiredChunkSizeReturnsCorrectValue)
+{
+    constexpr uint32_t PAYLOAD_SIZE{0U};
+    constexpr uint32_t PAYLOAD_ALIGNMENT{iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT};
+    constexpr uint32_t CUSTOM_HEADER_SIZE{iox::CHUNK_NO_CUSTOM_HEADER_SIZE};
+    constexpr uint32_t CUSTOM_HEADER_ALIGNMENT{iox::CHUNK_NO_CUSTOM_HEADER_ALIGNMENT};
+
+    constexpr uint32_t EXPECTED_SIZE{sizeof(ChunkHeader)};
+
+    auto sutResult =
+        ChunkSettings::create(PAYLOAD_SIZE, PAYLOAD_ALIGNMENT, CUSTOM_HEADER_SIZE, CUSTOM_HEADER_ALIGNMENT);
+    ASSERT_FALSE(sutResult.has_error());
+    auto& sut = sutResult.value();
+
+    EXPECT_THAT(sut.requiredChunkSize(), Eq(EXPECTED_SIZE));
+}
+
+// END GETTER METHOD TESTS
+
+// BEGIN EXCEEDING CHUNK SIZE TESTS
+
+TEST(ChunkSettings_test, NoCustomPayloadAlignmentAndTooLargePayload_Fails)
```

Comment:
Here the test names are lower case. I do not care much either way but it should be consistent within the file.